website/src/components/Explain.tsx

```diff
@@ -25,7 +26,10 @@ export const Explain = ({ explanation }: ExplainProps) => {
         <PopoverCloseButton />
         <PopoverBody>
           {explanation.map((paragraph, idx) => (
-            <Text key={idx}>{paragraph}</Text>
+            <>
```

Comment:
```suggestion
            <Fragment key={idx}>
```

Reply:
I tried changing `<></>` to `Fragment`, linter doesn't like it (1).
```tsx
          {explanation.map((paragraph, idx) =>
            <Fragment key={idx}>
              {paragraph}
              <br />
            </Fragment>
          )}
```

<img width="1134" alt="CleanShot 2023-02-05 at 17 51 14@2x" src="https://user-images.githubusercontent.com/24773652/216823347-04c8104e-dbe1-42af-9b95-fac5698f5afe.png">

If I remove `<br />`, it now doesn't like the `key` property (2)
<img width="1015" alt="CleanShot 2023-02-05 at 17 53 07@2x" src="https://user-images.githubusercontent.com/24773652/216823488-72e5523e-e1ec-48e9-836f-afeb355b6d3e.png">

If I remove `key` property, everything is fine again.

<img width="527" alt="CleanShot 2023-02-05 at 17 54 09@2x" src="https://user-images.githubusercontent.com/24773652/216823571-d14fef91-47e2-496c-bc07-b0e94d4b1823.png">

I'm not really sure what is going on here.

Reply:
hey @lainisourgod, sorry for the bad code suggestion, please try this:
```jsx
<Fragment key={idx}>
  <>
    {paragraph}
    <br />
  </>
</Fragment>
```

Reply:
Hmm, look like the TS doesn't complain when use change the type to ReactElement[]

Reply:
I'm not really proficient in TS type system, but using ReactElement here feels a hack, but if it's ok for your code base, then fine!